### ISSUE TITLE
Processes chapter

### ISSUE THREAD

**Issue by bebraw** [survivejs/maintenance-book#33]
Give this a pass and tweak where you feel it's necessary. Most likely it would be nice to discuss how it works with multiple people and different roles. That might use some image (I can draw if you do a rough one first).

**Comment by sapegin** [comment]
> Give this a pass and tweak where you feel it's necessary.

Done.

> Most likely it would be nice to discuss how it works with multiple people and different roles.

Any ideas for that?

> That might use some image (I can draw if you do a rough one first).

And for that ;-)

**Comment by bebraw** [comment]
>>Most likely it would be nice to discuss how it works with multiple people and different roles.

> Any ideas for that?

The key here might be to look into existing projects and show how they work or worked in the past. It doesn't have to be but enough to give inspiration and places to look at. One key thing to mention - Conway's law.

We can worry about the images later. It's easier once the content is solid and doesn't change so much.

**Comment by sapegin** [comment]
> The key here might be to look into existing projects and show how they work or worked in the past. 

Would you like to add that?

**Comment by bebraw** [comment]
> Would you like to add that?

Sure. I can add a few examples. Later on these could grow into case studies if there's time/interest towards that.

**Comment by sapegin** [comment]
Case studies would be very cool! I like how it’s done in GitHub’s guide: https://opensource.guide/best-practices/

**Comment by bebraw** [comment]
Yeah, I think it would be something we would provide for paying readers as bonus content. I have some ideas on how to approach this.
